src/k8s-engine/lima.ts

```diff
@@ -1092,6 +1099,37 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     }
   }
 
+  protected async startContainerd(): Promise<void> {
+    const workdir = await fs.promises.mkdtemp(path.join(os.tmpdir(), 'rd-containerd-install-'));
+
+    try {
+      const fixedProfile = path.join(workdir, 'profile');
+      const profileContents = (await fs.promises.readFile(resources.get('scripts', 'profile'))).toString();
+      const fixedContents = profileContents.replace(/export CONTAINERD_ADDRESS=.*/,
+        `export CONTAINERD_ADDRESS=${ CONTAINERD_ADDRESS_K3S }`);
+      const confListPath = path.join(workdir, '10-flannel.conflist');
+      const configPath = path.join(workdir, 'config.toml');
+
+      await fs.promises.writeFile(fixedProfile, fixedContents);
+      await this.lima('copy', fixedProfile, `${ MACHINE_NAME }:~/.profile`);
+
+      await fs.promises.writeFile(confListPath, FLANNEL_CONFLIST, { encoding: 'utf-8' });
+      await this.lima('copy', confListPath, `${ MACHINE_NAME }:/tmp/10-flannel.conflist`);
+      await this.ssh('sudo', 'mkdir', '-p', '/etc/cni/net.d');
+      await this.ssh('sudo', 'mv', '/tmp/10-flannel.conflist', '/etc/cni/net.d/10-flannel.conflist');
```

Comment:
Use `this.writeFile`, which handles all this temporary file + move gunk for you?

Reply:
Bump because comments were getting lost.

Reply:
> Use `this.writeFile`, which handles all this temporary file + move gunk for you?

`lima.ts` line 1134.
> await this.ssh('sudo', 'mv', '/tmp/10-flannel.conflist', '/etc/cni/net.d/10-flannel.conflist');

Reply:
That's been done.